### ISSUE TITLE
Change parsing of /used_for in getPublicationEvent on collections pages (from 2038)

### ISSUE THREAD

**Issue by gigamorph** [project-lux/lux-frontend#88]
**Problem Description:**
The event is not being parsed correctly on the collections pages.

**Expected Behavior/Solution:**
Fix it. TBD

**Requirements:**

1. Frontend needs to parse /used_for better (TBD)

**Needed for promotion:**
If an item on the list is not needed, it should be crossed off but not removed.

- [ ] Wireframe/Mockup - Mike
- [ ] Committee discussions - Sarah
- [ ] Feasibility/Team discussion - Sarah
- [ ] Backend requirements - TBD
- [ ] Frontend requirements- TBD
- [ ] Questions
How should the frontend be parsing the /used_for data point?

**UAT/LUX Examples:**

- https://lux-front-tst.collections.yale.edu/view/set/664c4aeb-532f-46a2-9db8-4d0b51c73a64

**Dependencies/Blocks:**
N/A

**Related Github Issues:**
N/A

**Related links:**

- Bugherd: https://www.bugherd.com/projects/284041/tasks/1975

**Wireframe/Mockup:**
Place wireframe/mockup for the proposed solution at end of ticket.

**Comment by roamye** [comment]
@kkdavis14 to follow up with Rob to see if this is still good to do.

**Comment by kkdavis14** [comment]
we can mark this low priority while Linked.art is sorting whether or not to do away with exhibition sets (needs community discussion).

**Comment by prowns** [comment]
@kamerynB  - can this be marked as unblocked?

**Comment by kamerynB** [comment]
@prowns I guess. I'll confirm with Kelly that the data is there and there isn't anything additional required on the backend.

**Comment by prowns** [comment]
Confirmed not blocked.

**Comment by prowns** [comment]
Ideally - the YCBA and YUAG sets would need to be merged in order to have an Exhibition set that can be used in this context.  Without this, there is no SET page that a user would encounter.   

Note - per Interactions, the separate links to YCBA exhibition set and YUAG exhibition set were removed from the right hand column since users could easily view all results and facet by repository.

#agenda for data

**Comment by clarkepeterf** [comment]
@kamerynB  do we want to add this to the next release because [ML #186](https://github.com/project-lux/lux-marklogic/issues/186) and [MT #40](https://github.com/project-lux/lux-middletier/issues/40) are completed?

**Comment by roamye** [comment]
The original UAT link DNE. It exists in PROD, but when I search for the link through SS it does not show up under works. I am wondering if the original TST link: https://lux-front-tst.collections.yale.edu/view/set/664c4aeb-532f-46a2-9db8-4d0b51c73a64 is now: 
https://lux-front-tst.collections.yale.edu/view/text/83020c94-2797-4944-8044-6a6396e63b4c ... but this isn't a set... 

if I put the original TST link into PROD the set record appears... but I also can't find this within the works simple searches in PROD or TST/DEV. 

@kamerynB - would you be able to help find the original link? or is this a bigger issue that needs a BH?

**Comment by prowns** [comment]
@roamye - tst is loading, so this should reappear.

**Comment by kkdavis14** [comment]
I guess if they can't ever land on the Set, they don't need to get back to the Event from a Set

**Comment by azaroth42** [comment]
You can't search for them either -- they're really just a join table at this point.
There could be other sets with events... but we can wait until that eventuates.

**Comment by roamye** [comment]
@prowns - from your comment should this not be deployed then as it is proposed-closed?

**Comment by roamye** [comment]
Approved by UAT 7/22

**Comment by roamye** [comment]
Looks good in PROD. Closing.
